backend/docker/sql/views/signed_extrinsics_per_day.sql

```diff
@@ -0,0 +1,11 @@
+CREATE VIEW signed_extrinsics_per_day AS
+SELECT *
+FROM (SELECT "when"::date FROM generate_series(timestamp '2021-10-16', timestamp '2022-11-17', interval  '1 day') "when") d
+LEFT JOIN (
+  SELECT to_timestamp(timestamp)::date AS "when", count(*) AS "volume"
+  FROM extrinsic
+  WHERE is_signed=true
+  GROUP BY to_timestamp(timestamp)::date
```

Comment:
Let's declare `to_timestamp(timestamp)` once.

Reply:
Can't use variables in psql.